agent/agent.go

```diff
@@ -303,7 +308,23 @@ func (a *Agent) runPostTaskCommands(ctx context.Context, tc *taskContext) {
 		} else {
 			tc.logger.Task().Infof("Finished running post-task commands in %v.", time.Since(start).String())
 		}
-		a.killProcs(tc)
+		grip.Info("Finished running post-task commands")
+	}
+}
+
+func (a *Agent) runPostGroupCommands(ctx context.Context, tc *taskContext, taskGroup *model.TaskGroup) {
+	grip.Info("Running post-group commands")
+	a.killProcs(tc)
+	defer a.killProcs(tc)
```

Comment:
I think we need to kill procs inbetween tasks even after task groups run. 

we can make this configurable now or later.

Reply:
Sharing procs is EVG-2671. If you're saying that we should kill procs between tasks even if postGroupTaskCommands doesn't run, this is handled already by `runTask`.